_includes/why_d4a.md

```diff
@@ -0,0 +1,113 @@
+Docker swarm mode on these platforms.  When the cluster is finished booting, you
+can jump right in and start running `docker service` commands.
+
+We also provide a prescriptive upgrade path that is intended to help users
+upgrade between various versions of Docker for {{cloudprovider}} and the Docker
+Engine itself in a smooth and automatic way.  Instead of having "maintenance
+dread" set in as you ponder your future responsibilities upgrading the software
+you are using, these product will allow you to quickly and simply upgrade to new
+versions when they are released.
+
+## Minimal, Docker-Focused Base
+
+The custom Linux distribution Docker for {{cloudprovider}} is built on top of is
+painstakingly configured and developed to run Docker in the most ideal manner
+possible.  Everything from the kernel configuration to the networking stack has
+been customized to make it a favorable place for running Docker.  For instance,
+we have ensured that the kernel versions are compatible with the latest and
+greatest in Docker functionality, such as the `overlay2` storage driver.
+
+Instead of facing the tradeoffs that a general purpose operating system makes,
+the Linux distribution that these projects are built on top of is freed to focus
+only on one thing:  Providing the best _Docker_ experience to you and your team.
+
+## Self-Cleaning and Self-Healing
+
+Even the most conscientious admin can be caught off guard by issues such as
+unexpectedly aggressive logging or the Linux kernel killing memory-hungry
+processes.  In Docker for {{cloudprovider}}, the cluster you receive to run
+applications on is resilient to a variety of such issues by default.
+
+Log rotation native to the host is configured for you automatically, ensuring
+chatty logs won't use up all of your disk space.  Likewise, the "system prune"
+option allows you to ensure unused Docker resources such as old images are
+cleaned up automatically.  The lifecycle of nodes is managed using auto-scaling
+groups or similar constructs, so that if a Docker daemon dies or a node goes
+sideways for unforeseen reasons, the node will be taken out of load balancer
+rotation and/or replaced automatically and all of its container tasks will be
+rescheduled.
+
+These self-cleaning and self-healing properties work out of the box without
+requiring configuration or input from the user, and will allow you to breathe
+easier as the risk of downtime is reduced.
+
+## Logging Native to the Platforms
+
+Centralized logging is a critical component of many modern infrastructure
+stacks.  To have these logs indexed and searchable proves invaluable for
+debugging appliation and system issues as they come up.  Out of the box, Docker
+for {{cloudprovider}} will forward logs from containers to a native cloud
+provider abstraction ({{cloudprovider_log_dest}}).
+
+## Next-Generation Docker Bug Reporting Tools
+
+One of the most common pains in open source issue reporting is effectively
```

Comment:
This is about communicating information to upstream maintainers though, after you've identified issues already.